### PR TITLE
Docs: document default cert annotation.

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

- First commit alphabetizes table of annotations.
- ~Second commit adds `ingress.pomerium.io/default-cert-secret` to a tooltip, pointing to the annotations documentation in the project README (staged)~.
- Third commit undoes the second, and documents defining a default cert as a Helm configuration option.

## Related issues

Documents https://github.com/pomerium/ingress-controller/pull/151
Blocked by https://github.com/pomerium/pomerium-helm/pull/276

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review - Tagging @wasaga who created the annotation.

### CODE REVIEW

File: docs/docs/k8s/ingress.md
Comment (wasaga) [inline docs/docs/k8s/ingress.md:165]: this is not an `Ingress` resource annotation. This is an annotation for `IngressClass`, that allows you to skip providing `Ingress.spec.TLS.secret`

**Review by wasaga — CHANGES_REQUESTED** [review]

File: docs/docs/k8s/ingress.md
Comment (alexfornuto) [reply docs/docs/k8s/ingress.md:165]: @wasaga thanks for catching this. Where would a user apply it, and where should we document it?

Comment (wasaga) [reply docs/docs/k8s/ingress.md:165]: `IngressClass` is currently not documented as it is created by the Helm chart. 

The spec is rather simple: 
```yaml
apiVersion: networking.k8s.io/v1
kind: IngressClass
metadata:
  annotations:
    ingress.pomerium.io/default-cert-secret: pomerium/wildcard-localhost-pomerium-io
  name: pomerium
spec:
  controller: pomerium.io/ingress-controller
```

I suppose here we should document the existence of such annotation on the ingress class, and refer the reader to the Helm documentation reference for further detail.

File: docs/docs/k8s/ingress.md
Comment (wasaga) [inline docs/docs/k8s/ingress.md:248]: why refer to github, why not put it here for consistency.

Comment (alexfornuto) [reply docs/docs/k8s/ingress.md:248]: Because, as you noted before, we don't document defining the IngressClass anywhere else in docs since it's created by the Helm chart, and our docs are based around an installation with Helm.
In other words, documenting this annotation in the docs would also mean creating a new section, possibly a new page, on the IngressController class.

With that said, does this decision make sense, or should we discuss implementing such a section/page?

Comment (wasaga) [reply docs/docs/k8s/ingress.md:248]: I believe we should document Helm option that sets this annotation for a chart. 
However it also makes sense to describe this annotation in the IngressClass separate section.

Comment (alexfornuto) [reply docs/docs/k8s/ingress.md]: Moved to a different doc in a different form.

**Review by travisgroth — APPROVED** [review]
LGTM just pending https://github.com/pomerium/pomerium-helm/pull/276, which is waiting on ingress controller release.

**Pull request merged**
